recipes/mmcv/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set version = "1.5.3" %}
+{% set number = 0 %}
+
+{% if cuda_compiler_version != "None" %}
+{% set number = number + 200 %}
+{% endif %}
+
+package:
+  name: mmcv-recipe
+  version: {{ version }}
+
+source:
+  url: https://github.com/open-mmlab/mmcv/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 650929f52f1d0b2bae1bdaaa874e1e6dfdfa15f40bab5c366330488a9949ddd0
+build:
+  number: {{ number }}
+  skip: true  # [win]
+
+outputs:
+  - name: mmcv-full
```

Comment:
Yeah, that's how vendor distributes it:

```
mmcv-full: comprehensive, with full features and various CUDA ops out of box. It takes longer time to build.

mmcv: lite, without CUDA ops but all other features, similar to mmcv<1.0.0. It is useful when you do not need those CUDA ops.
```
https://mmcv.readthedocs.io/en/latest/get_started/installation.html

Pip packages are also `mmcv` and `mmcv-full`. The former one is very light and pure-python, so it will be easy to add it later.